Description: fix includes path
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/intro_c.c
+++ b/intro_c.c
@@ -68,7 +68,7 @@
 #define oglActiveTexture glActiveTexture
 #endif
 
-#include "glext.h"
+#include <GL/glext.h>
 
 #include "4klang.h"
 
